server/lib/pbench/indexer.py

```diff
@@ -22,11 +21,11 @@
 from collections import Counter, deque
 from configparser import ConfigParser, Error as ConfigParserError, \
         NoSectionError, NoOptionError
```

Comment:
linebreak, just to make it easier

Reply:
Let's leave that to a formatting commit.  This change only addresses when is reported by `flake8`.  I'd rather let the forthcoming `black` reformatting handle this.